<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout"/>
    </item>
    <item row="2" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="getGPA">
        <property name="styleSheet">
         <string notr="true">QPushButton {
min-height: 30px;
}</string>
        </property>
        <property name="text">
         <string>Calculate GPA</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>GPA: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="GPA">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="addCourse">
        <property name="minimumSize">
         <size>
          <width>386</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
min-height: 30px;
}</string>
        </property>
        <property name="text">
         <string>Add Course</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeCourse">
        <property name="styleSheet">
         <string notr="true">QPushButton {
min-height: 30px;
}</string>
        </property>
        <property name="text">
         <string>Remove Course</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
